<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDetail</class>
 <widget class="QDialog" name="DialogDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seam allowance tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../vmisc/share/resources/icon.qrc">
    <normaloff>:/icon/64x64/icon64x64.png</normaloff>:/icon/64x64/icon64x64.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <widget class="QLabel" name="helpLabel">
       <property name="geometry">
        <rect>
         <x>59</x>
         <y>468</y>
         <width>504</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Ready!</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>367</width>
         <height>34</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7"/>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>561</width>
         <height>361</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Bias X:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBoxBiasX">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <double>-900.990000000000009</double>
              </property>
              <property name="maximum">
               <double>900.990000000000009</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelUnitX">
              <property name="text">
               <string>cm</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Bias Y:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBoxBiasY">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <double>-900.990000000000009</double>
              </property>
              <property name="maximum">
               <double>900.990000000000009</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelUnitY">
              <property name="text">
               <string>cm</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QCheckBox" name="checkBoxReverse">
              <property name="text">
               <string>Reverse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Options</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout"/>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxSeams">
               <property name="text">
                <string>Seam allowance</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="labelEditWidth">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="locale">
                  <locale language="English" country="UnitedStates"/>
                 </property>
                 <property name="text">
                  <string>Width:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBoxSeams">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximum">
                  <double>900.990000000000009</double>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                 <property name="value">
                  <double>1.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelUnit">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>cm</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxClosed">
               <property name="text">
                <string>Closed</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_16">
               <property name="sizeConstraint">
                <enum>QLayout::SetFixedSize</enum>
               </property>
               <property name="leftMargin">
                <number>10</number>
               </property>
               <property name="topMargin">
                <number>10</number>
               </property>
               <property name="rightMargin">
                <number>10</number>
               </property>
               <property name="bottomMargin">
                <number>10</number>
               </property>
               <item alignment="Qt::AlignLeft">
                <widget class="QToolButton" name="toolButtonDelete">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Delete</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item alignment="Qt::AlignRight">
                <widget class="QToolButton" name="toolButtonDown">
                 <property name="toolTip">
                  <string>Scroll down the list</string>
                 </property>
                 <property name="text">
                  <string notr="true">...</string>
                 </property>
                 <property name="icon">
                  <iconset theme="go-down">
                   <normaloff/>
                  </iconset>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignRight">
                <widget class="QToolButton" name="toolButtonUp">
                 <property name="toolTip">
                  <string>Scroll up the list</string>
                 </property>
                 <property name="text">
                  <string notr="true">...</string>
                 </property>
                 <property name="icon">
                  <iconset theme="go-up">
                   <normaloff/>
                  </iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>0</y>
         <width>327</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>All objects in path should follow in clockwise direction.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>32</width>
         <height>32</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../vmisc/share/resources/icon.qrc">:/icon/32x32/clockwise.png</pixmap>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Pattern piece data</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <widget class="QListWidget" name="listWidgetMCP">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>0</y>
            <width>201</width>
            <height>381</height>
           </rect>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>70</y>
            <width>271</width>
            <height>153</height>
           </rect>
          </property>
          <property name="title">
           <string>Material/Cut number/Placement</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Material type:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" colspan="2">
            <widget class="QComboBox" name="comboBoxMaterial">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Cut number:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2" colspan="2">
            <widget class="QSpinBox" name="spinBoxCutNumber">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Placement:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2" colspan="2">
            <widget class="QComboBox" name="comboBoxPlacement"/>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="pushButtonAdd">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QPushButton" name="pushButtonCancel">
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="pushButtonRemove">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="formLayoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>271</width>
            <height>61</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Letter:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditLetter">
             <property name="maxLength">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelEditName">
             <property name="text">
              <string>Name of detail:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditName">
             <property name="maxLength">
              <number>15</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="formLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>230</y>
            <width>261</width>
            <height>80</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBoxDetail">
             <property name="text">
              <string>Detail label visible</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBoxPattern">
             <property name="text">
              <string>Pattern label visible</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditLetter</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>comboBoxMaterial</tabstop>
  <tabstop>spinBoxCutNumber</tabstop>
  <tabstop>comboBoxPlacement</tabstop>
  <tabstop>pushButtonAdd</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonRemove</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>listWidgetMCP</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>toolButtonUp</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>checkBoxClosed</tabstop>
  <tabstop>toolButtonDelete</tabstop>
  <tabstop>doubleSpinBoxBiasX</tabstop>
  <tabstop>doubleSpinBoxBiasY</tabstop>
  <tabstop>checkBoxReverse</tabstop>
  <tabstop>checkBoxSeams</tabstop>
  <tabstop>doubleSpinBoxSeams</tabstop>
  <tabstop>toolButtonDown</tabstop>
 </tabstops>
 <resources>
  <include location="../../../vmisc/share/resources/icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogDetail</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogDetail</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
